config/coreboot-novacustom-v560tu.config

```diff
@@ -0,0 +1,961 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# coreboot configuration
+#
+
+#
+# General setup
+#
+CONFIG_COREBOOT_BUILD=y
+CONFIG_LOCALVERSION=""
```

Comment:
Rebranding related. Maybe Dasharo wants to fixate this so LOCALVERSION is defined across all branding, but otherwise branding related snippets to understand:


Heads branding if not defined:
https://github.com/linuxboot/heads/blob/99157f22919d5c6d4cf21d1aa3c2fd174d0d9f13/Makefile#L13

Override by localsite
https://github.com/linuxboot/heads/blob/99157f22919d5c6d4cf21d1aa3c2fd174d0d9f13/Makefile#L92-L97

And then if not defined, populated
https://github.com/linuxboot/heads/blob/99157f22919d5c6d4cf21d1aa3c2fd174d0d9f13/modules/coreboot#L115-L119